ai2modulus/models/fno/fno.py

```diff
@@ -15,8 +15,8 @@
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
-import modulus.models.layers as layers
-import modulus
+import ai2ai2modulus.models.layers as layers
```

Comment:
Replace happened twice. Would be worth doing a grep to check for other instances.

Reply:
Fixed. No others according to my grep.